src/components/Project/ProjectHashTags/HashTagsContainer.tsx

```diff
@@ -12,8 +12,11 @@ interface IHashTagsProps {
 
 const HashTagsContainer: FC<IHashTagsProps> = ({ tags, onClick, onDelete, id }) => {
   const { t } = useTranslation();
-  const getAriaLabel = (tag: string) =>
-    t(onDelete ? `removeHashTag ${tag}` : onClick ? `addHashTag ${tag}` : '');
+  const getAriaLabel = (tag: string) => {
```

Comment:
move this utility function outside the component

Reply:
fixed